bob-backend/src/pearl/disk_controller.rs

```diff
@@ -400,10 +400,10 @@ impl DiskController {
     }
 
     pub(crate) async fn put(
-        &self,
+        self: Arc<Self>,
         op: Operation,
         key: BobKey,
-        data: &BobData,
+        data: BobData,
```

Comment:
You can still pass `&BobData`. And I think this is better in general

Reply:
Same situation as with self, it is not possible to pass simple reference to another thread (in case of timeout this reference will expire)

Reply:
Ok, it is possible in case of async block, so I will use it